src/components/analytics/index.ts

```diff
@@ -1,94 +0,0 @@
-import { useWeb3React } from '@web3-react/core'
-import { useEffect } from 'react'
-import { UaEventOptions } from 'react-ga4/types/ga4'
-import { useLocation } from 'react-router-dom'
-import { isMobile } from 'utils/userAgent'
-import { getCLS, getFCP, getFID, getLCP, Metric } from 'web-vitals'
-
-import GoogleAnalyticsProvider from './GoogleAnalyticsProvider'
-
-const GOOGLE_ANALYTICS_CLIENT_ID_STORAGE_KEY = 'ga_client_id'
-const GOOGLE_ANALYTICS_ID: string | undefined = process.env.REACT_APP_GOOGLE_ANALYTICS_ID
-
-const storedClientId = window.localStorage.getItem(GOOGLE_ANALYTICS_CLIENT_ID_STORAGE_KEY)
-
-const googleAnalytics = new GoogleAnalyticsProvider()
-
-export function sendEvent(event: string | UaEventOptions, params?: any) {
-  return googleAnalytics.sendEvent(event, params)
-}
-
-export function outboundLink(
```

Comment:
Do we want to capture outboundLink's through Amplitude?

Reply:
Could you ticket this, linking this PR, and then reach out to product to see if they still want this as a metric?
In case they do, make sure this PR is linked to from the ticket, so we can recover the relevant code without re-implementing (eg anonymizeLink, outboundLink).